search/mongo_search_test.go

```diff
@@ -205,18 +233,20 @@ func (m *MongoSearchSuite) TestImagingStudyBodySiteQueryObjectBySystemAndCode(c
 
 func (m *MongoSearchSuite) TestImagingStudyBodySiteQueryBySystemAndCode(c *C) {
 	q := Query{"ImagingStudy", "bodysite=http://snomed.info/sct|67734004"}
-	mq := m.MongoSearcher.CreateQuery(q)
-	num, err := mq.Count()
+
+	results, _, err := m.MongoSearcher.Search(q)
 	util.CheckErr(err)
-	c.Assert(num, Equals, 1)
+	resultsVal := reflect.ValueOf(results).Elem()
+	c.Assert(resultsVal.Len(), Equals, 1)
```

Comment:
similar to my last comment, this could become a re-usable function:
```
func (m *MongoSearchSuite) assertResultLen(results interface{}, expectedLen int)
```